File: tests/fixtures/typescript/excess_args.ts
function createUser(
    firstName: string,
    lastName: string,
    email: string,
    phone: string,
    address: string,
    city: string,
    state: string,
    zipcode: string
): any {
    return { firstName, lastName, email, phone, address, city, state, zipcode };
}

function simpleFunc(a: number, b: number): number {
    return a + b;
}

class UserService {
    private db: any;
    private cache: any;
    private logger: any;
    private mailer: any;
    private validator: any;
    private scheduler: any;

    constructor(db: any, cache: any, logger: any, mailer: any, validator: any, scheduler: any) {
        this.db = db;
        this.cache = cache;
        this.logger = logger;
        this.mailer = mailer;
        this.validator = validator;
        this.scheduler = scheduler;
    }

    getUser(userId: string): any {
        return this.db.get(userId);
    }
}

File: tests/fixtures/python/excess_args.py
"""Functions with too many parameters."""


def create_user(first_name, last_name, email, phone, address, city, state, zipcode):
    return {
        "first_name": first_name,
        "last_name": last_name,
        "email": email,
        "phone": phone,
        "address": address,
        "city": city,
        "state": state,
        "zipcode": zipcode,
    }


def simple_func(a, b):
    return a + b


class UserService:
    def __init__(self, db, cache, logger, mailer, validator, scheduler, p7, p8, p9):
        self.db = db
        self.cache = cache
        self.logger = logger
        self.mailer = mailer
        self.validator = validator
        self.scheduler = scheduler

    def get_user(self, user_id):
        return self.db.get(user_id)
